bucket/filezilla.json

```diff
@@ -15,10 +15,15 @@
     },
     "extract_dir": "FileZilla-3.43.0",
     "pre_install": [
-        "$file = 'fzdefaults.xml'",
-        "if (-not (Test-Path \"$persist_dir\\$file\")) {",
-        "    Copy-Item \"$dir\\docs\\fzdefaults.xml.example\" \"$dir\\fzdefaults.xml\"",
-        "}"
+        "@\"",
+        "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\" ?>",
+        "<FileZilla3>",
+        "  <Settings>",
+        "    <Setting name=\"Config Location\">$persist_dir</Setting>",
```

Comment:
Create subfolder and then persist this folder

Reply:
Yes, done